README.md

```diff
@@ -1 +1,95 @@
-# cars-bnb-frontend
\ No newline at end of file
+# Cars BnB App
```

Comment:
- [ ] There was no open PR for your backend.  I tried to review it but this happens when I try to create the database:

![image](https://user-images.githubusercontent.com/57421823/200096574-01f1dcbd-d4e7-4b9d-9741-6229cc0f9894.png)

This means I couldn't review both the backend and the frontend functionality because there was no API running to consume.  Also, I wasn't able to run the tests (because I couldn't create the db).

On the frontend, I was able to get to the initial screen, but not any further.

If this is a bug, kindly fix it.  If this is not a bug and your app requires something extra not mentioned in your instructions, kindly add it.

Kindly ensure the app works (both backend and frontend) to make a proper review on the functionality.

Since there is no PR for the backend, I'll leave my comments here in this review.